reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md

```diff
@@ -0,0 +1,158 @@
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Invoke-DscResource` cmdlet runs a method of a specified PowerShell Desired State Configuration
+(DSC) resource.
+
+This cmdlet invokes a DSC resource directly, without creating a configuration document.
+Using this cmdlet, configuration management products can manage windows or Linux by using DSC resources.
+This cmdlet also enables debugging of resources when the DSC engine is running with debugging enabled.
+
+## EXAMPLES
+
+### Example 1: Invoke the Set method of a resource by specifying its mandatory properties
+
+This command invokes the **Set** method of a resource named Log and specifies a **Message** property
+for it.
+
+```powershell
+PS C:\> Invoke-DscResource -Name Log -Method Set -Property @{Message = 'Hello World'} -ModuleName PSDesiredStateConfiguration
+```
+
+### Example 2: Invoke the Test method of a resource for a specified module
+
+This command invokes the **Test** method of a resource named WindowsProcess, which is in the module
+named PSDesiredStateConfiguration.
+
+```powershell
+PS C:\> Invoke-DscResource -Name WindowsProcess -Method Test -Property @{Path = 'C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe'; Arguments = ''} -ModuleName PSDesiredStateConfiguration
+```
+
+## PARAMETERS
+
+### -Method
+
+Specifies the method of the resource that this cmdlet invokes. The acceptable values for this
+parameter are: **Get**, **Set**, and **Test**.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+Accepted values: Get, Set, Test
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ModuleName
+
+Specifies the name of the module from which this cmdlet invokes the specified resource.
+
+```yaml
+Type: ModuleSpecification
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Name
+
+Specifies the name of the DSC resource to start.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Property
+
+Specifies the resource property name and its value in a hash table as key and value, respectively.
+
+```yaml
+Type: Hashtable
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see
+[about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

Comment:
this should be:

This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, 
`-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, 
`-WarningAction`, and `-WarningVariable`. For more information, see 
[about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

Reply:
This change is not needed.